addons/mail/tests/test_mail_message_format.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo.tests.common import HttpCase, tagged, new_test_user
+
+
+@tagged("post_install")
+class TestMailMessageFormat(HttpCase):
```

Comment:
This test should not be done in mail, as it depends on livechat; morover I doubt it is really usefull (what you do is only preventing people from removing 'user_livechat_username' and I don't think any test is necessary: either you remove it and you remove the test, either you keep it). Not sure to see how it prevents any regression.